src/features/profile/DangerCard.js

```diff
@@ -15,6 +15,9 @@ import { CardContentSection } from "../../components"
 import { selectAllEntries, deleteAllEntries } from "../entries"
 import { useSettings } from "../settings"
 
+import { FATHOM_DELETE_ALL_MENSES_TAGS, FATHOM_DELETE_TAGS_CONFIRMED } from "../../constants";
```

Comment:
**nitpick (non-blocking):** I like to have the imports with more `..` above the once with less.